src/main/java/com/wanderersoftherift/wotr/datagen/WotrBlockLootTableProvider.java

```diff
@@ -33,6 +34,25 @@ protected void generate() {
         dropSelf(WotrBlocks.MOB_TRAP_BLOCK.get());
         dropSelf(WotrBlocks.SPRING_BLOCK.get());
         add(WotrBlocks.RIFT_MOB_SPAWNER.get(), noDrop());
+        add(WotrBlocks.NOGRAVGRAVEL.get(), createSingleItemTable(Blocks.GRAVEL));
```

Comment:
Update the loottable to add the riftcheck.

Reply:
Requires addition of a proper Rift LootItemCondition check See RiftLevelCheck.

Reply:
What do you have in mind Pat - I would have thought NoGravGravel should drop normal gravel regardless of dimension?

Reply:
If we allow the crafting and building in overworld, it is going to drive me insane. I often redo all my blocks, just break them down and rebuild them. With this I would have to continuously recraft the no gravs.

We do want it to drop as normal blocks in rifts, otherwise there is no way to get the normal blocks from rifts.

Reply:
Ah, ok, makes sense.